src/osd/OSD.cc

```diff
@@ -1479,6 +1479,45 @@ class PGOpQueueable : public OpQueueItem::OpQueueable {
   }
 };
 
+class PGOpQueueableNoLock : public OpQueueItem::OpQueueable {
```

Comment:
This is the way to change the locking behavior, but simply removing the lock and calling the same code obviously is wrong...